### PR TITLE
ADD: disconnect by peer id (ios)

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: ios/RnLdk.m [hunk 1]
```diff
@@ -28,10 +28,13 @@ @interface RCT_EXTERN_MODULE(RnLdk, NSObject)
                   resolve:(RCTPromiseResolveBlock)resolve reject:(RCTPromiseRejectBlock)reject)
 
 RCT_EXTERN_METHOD(connectPeer:(NSString *)pubkeyHex
-                  hostname:(NSString *)pubkeyHex
+                  hostname:(NSString *)hostname
```
[the remark this answers is not in the document]
Comment (marcosrdz) [reply ios/RnLdk.m:31]: Why the name change here and not on Swift?

[the remark this answers is not in the document]
Comment (Overtorment) [reply ios/RnLdk.m:31]: im confused because there is already  one pubkeyhex, on line 30

**Review by marcosrdz — APPROVED** [review]

File: ios/RnLdk.m (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcosrdz) [reply ios/RnLdk.m:31]: my bad. i was looking at the wrong file

[the remark this answers is not in the document]
Comment (Overtorment) [reply ios/RnLdk.m:31]: how did it work before?

**Pull request merged**